tests/robot-tests/tests/admin/bau/comments.robot

```diff
@@ -0,0 +1,199 @@
+    user waits until parent contains element    ${details}    xpath:.//a[text()="Edit this release"]
+    user clicks link    Edit this release
+
+Navigate to content section as bau1
+    [Tags]    HappyPath
+    user clicks link    Content
+    user closes Set Page View box
+
+Add review comment for first content block
+    [Tags]    HappyPath
+    user scrolls to element    //*[@data-testid="accordionSection"]
+    user clicks element    //*[@type="button" and text()="First content section"]
+
+    user scrolls to element    //*[@data-testid="Expand Details Section Add / View comments (0 unresolved)"]
+
+    user clicks element    //*[@data-testid="Expand Details Section Add / View comments (0 unresolved)"]
+
+    # avoid set page view box getting in the way 
+    user scrolls down  400
+
+    user presses keys    This section needs fixing    //*[@data-testid="comment-textarea"]
+    user clicks button    Add comment
+
+    user waits until page contains element
+    ...    //*[@data-testid="comment-content" and text()="This section needs fixing"]    5
+    user clicks element    //*[@data-testid="Expand Details Section Add / View comments (1 unresolved)"]
+
+Switch to analyst1 and navigate to release
+    [Tags]    HappyPath
+    user changes to analyst1
+    user selects theme and topic from admin dashboard    %{TEST_THEME_NAME}    %{TEST_TOPIC_NAME}
+    user opens publication on the admin dashboard    ${PUBLICATION_NAME}
+    ${accordion}=    user gets accordion section content element    ${PUBLICATION_NAME}
+    user opens details dropdown    ${RELEASE_NAME}    ${accordion}
+    ${details}=    user gets details content element    ${RELEASE_NAME} (not Live)    ${accordion}
+    user waits until parent contains element    ${details}    xpath:.//a[text()="Edit this release"]
+    user clicks link    Edit this release
+
+Navigate to content section to resolve comments
+    [Tags]    HappyPath
+    user clicks link    Content
+    user waits until h1 is visible    ${PUBLICATION_NAME}
+
+Resolve comments
+    [Tags]    HappyPath
+    user closes Set Page View box
+    user scrolls to element    //*[@data-testid="accordionSection"]
+    user clicks element    //*[@type="button" and text()="First content section"]
+
+    user scrolls to element    //*[@data-testid="Expand Details Section Add / View comments (1 unresolved)"]
+
+    user waits until element is visible    //*[@data-testid="Expand Details Section Add / View comments (1 unresolved)"]
+    user clicks element    //*[@data-testid="Expand Details Section Add / View comments (1 unresolved)"]
+    user scrolls down  400
+    user presses keys    I've fixed this issue    //*[@data-testid="comment-textarea"]
+    user clicks button    Add comment
+
+    # avoid set page view box intercepting clicks
+    user scrolls to element    //*[@data-testid="resolve-button"]
+    user waits until element is enabled    //*[@data-testid="resolve-button"]
+    user clicks button    Resolve
```

Comment:
At this point there are two `Resolve` buttons. Maybe we could do something like this?
```
${comment1}=  user gets comment   Comment text here
user clicks button  Resolve   ${comment1}
```

Where `user gets comment` grabs the `li` that contains that comment-content

Not sure if this is worth the effort though - so could just leave this.